install_config/persistent_storage/topics/glusterfs_overview_block_volumes.adoc

```diff
@@ -14,6 +14,5 @@ improvements for certain workloads.
 
 [IMPORTANT]
 ====
-At this time, it is recommended to only use gluster-block volumes for OpenShift
-Logging and OpenShift Metrics storage.
+For more information about OpenShift Container Storage and OpenShift Container Platform interoperability, see link: https://access.redhat.com/articles/3403951[OpenShift Container Storage and OpenShift Container Platform interoperability matrix].
```

Comment:
Small nit: we use attribute `{product-title} instead of `OpenShift Container Platform`, would you mind changing this?

Reply:
@bobfuru : thank you for reviewing this.
I have replaced Openshift Container Platform with {product-title} and squashed it to this previous commit. (This is my first time squashing and looks like it has worked) Please let me know if there is any problem with this commit. Thank you!